homing_missiles.py

```diff
@@ -69,7 +71,18 @@ def update(self, events):
         for event in events:
             if event.type == pygame.KEYDOWN:
                 if event.key == EVENT_KEY_ENTER:
-                    target_location = self.get_target_position(self.current_text_)
+                    target_id = self.get_target_id(self.current_text_)
+                    if target_id and self.current_text_ != "fire":
+                        if len(self.targetted_enemies_) == 5:
+                            untargetted_enemy = self.targetted_enemies_.pop(0)
+                        self.targetted_enemies_.append(self.universe_.enemies_[target_id])
+                       
+                    if len(self.targetted_enemies_) > 0 and self.current_text_ == "fire":
```

Comment:
So what happens if there's an enemy named fire? My solution would be to remove `fire` from our dictionary, we'll count it as a keyword for us.